bboxes_labelling/run_additional_channel.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+#SBATCH -p gpu # partition
+#SBATCH -N 1   # number of nodes
+#SBATCH --mem 8G # memory pool for all cores
+#SBATCH -n 2 # number of cores
+#SBATCH -t 3-00:00 # time (D-HH:MM)
+#SBATCH --mail-type=ALL
+#SBATCH --mail-user=n.aznan@ucl.ac.uk
+
+# ---------------------
+# Load required modules
+# ----------------------
+module load SLEAP
```

Comment:
I think we don't need SLEAP here, right? 

(`module load` here acts as a sort of activating a virtual environment for SLEAP and its dependencies, see more [here](https://www.bu.edu/tech/support/research/software-and-programming/software-and-applications/modules/)